### PR TITLE
Notebooks: makes notebooks deterministic and prints output of asserts

### PR DESCRIPTION
**Pull request by fabianandresgrob**
PR for #842 and #843

### CODE REVIEW

**Comment by fabianandresgrob** [comment]
Checked using that script, everything looks good. Only minor things differ like kernel id. I added one more `torch.manual_seed()` to the ONNX export.

**Comment by Giuseppe5** [comment]
When doing multiple runs to compare, restart the kernels, just to be sure

File: notebooks/02_quant_activation_overview.ipynb [hunk 1]
```diff
@@ -315,16 +326,27 @@
...
+    }
+   ],
    "source": [
     "return_disabled_quant_relu = QuantReLU(act_quant=None, return_quant_tensor=True)\n",
     "relu_out_tensor = return_disabled_quant_relu(out_tensor)\n",
-    "assert relu_out_tensor.is_valid==True\n",
-    "assert relu_out_tensor.scale == out_tensor.scale\n",
-    "assert relu_out_tensor.zero_point == out_tensor.zero_point\n",
-    "assert relu_out_tensor.bit_width == out_tensor.bit_width"
+    "assert_with_message((relu_out_tensor.is_valid==True))\n",
```
Comment (Giuseppe5) [inline notebooks/02_quant_activation_overview.ipynb:346]: Double parenthesis

Comment (Giuseppe5) [reply notebooks/02_quant_activation_overview.ipynb:346]: Also, let's remove `assert condition == True`, and change it to `assert condition`.
Similar with `assert condition == False` that becomes `assert not condition`
This line becomes :
```suggestion
    "assert_with_message(relu_out_tensor.is_valid)\n",
```
I think there are other instances around the notebooks

**Review by nickfraser — APPROVED** [review]
LGTM!

**Pull request merged**